src/meta/expr.lean

```diff
@@ -326,6 +326,16 @@ meta def nat.to_pexpr : ℕ → pexpr
 | 0 := ``(0)
 | 1 := ``(1)
 | n := if n % 2 = 0 then ``(bit0 %%(nat.to_pexpr (n/2))) else ``(bit1 %%(nat.to_pexpr (n/2)))
```

Comment:
Is this different to
https://github.com/leanprover-community/lean/blob/68455b087d87e9dc3f736da0de95807e05260460/library/init/meta/has_reflect.lean#L31-L35